src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +119,25 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.ofHours(TIME_SHIFT));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        return date.matches(REGEX1)
```

Comment:
let's remove check on Regex and just catch exception and return Optional.empty() after catch block

Reply:
done